Core/Utilities/Graphs/Graph.py

```diff
@@ -334,5 +334,3 @@ def writeGraph( self, fname, fileFormat = 'PNG' ):
     self.canvas.draw()
     if fileFormat.lower() == 'png':
       self.canvas.print_png(fname)
-    elif fileFormat.lower() == 'svg':
```

Comment:
Could you please add an error message?
```python
if fileFormat.lower() == 'png':
  self.canvas.print_png(fname)
else:
  gLogger.error("%s is not supported file format" %fileFormat)
```
In case someone will modify it will get an error message. Thanks.

Reply:
Done